scripts/screeps_rl_simulator_harness.py

```
@@ -5924,6 +5926,123 @@ def _safe_redact_smoke_payload(payload: Any) -> JsonObject:
+                "attempts": attempts,
+                "maxAttempts": max_attempts,
+                "retrySeconds": retry_seconds,
+                "nextAction": (
+                    "inspect private-simulator reset/import room state; do not rerun paid validation unchanged "
+                    "until the room-busy placement lock is explained"
+                ),
+            }
+            raise RuntimeError(
+                f"place-spawn room busy after {max_attempts} attempt(s): "
+                f"{_safe_redact_smoke_payload(detail)}"
+            )
```

Ruff (0.15.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)